### ISSUE TITLE
Find all refs doesn't find field/prop used in complex pattern member.

### ISSUE THREAD

**Issue by CyrusNajmabadi** [dotnet/roslyn#9073]
Given the following:

```c#
    public class Expression { }
    public class Constant : Expression
    {
        public readonly int Value;
        public Constant(int Value)
        {
            this.Value = Value;
        }
    }
    public class Plus : Expression
    {
        public readonly Expression $$Left, Right;   // Find All Refs on 'Left'
        public Plus(Expression Left, Expression Right)
        {
            this.Left = Left;
            this.Right = Right;
        }
    }
    public class X
    {
        public static void Main()
        {
            Expression expr = null;
            if (expr is Plus { Left is Plus })
```

Find all Refs on Plus.Left does not find the match in the complex pattern.

**Comment by jaredpar** [comment]
Reopen pending unit tests.

**Comment by gafter** [comment]
Fixed in features/patterns via #9439.
